src/main/java/teammates/common/datatransfer/InstructorPrivileges.java

```diff
@@ -271,8 +271,8 @@ private void updatePrivilegesInSessionLevel(String sectionName, String sessionNa
     private void verifyExistenceOfsectionName(String sectionName) {
         if (!this.sessionLevel.containsKey(sectionName)) {
             addSectionWithDefaultPrivileges(sectionName);
-            this.sessionLevel.put(sectionName, new LinkedHashMap<String, Map<String, Boolean>>());
         }
+        this.sessionLevel.putIfAbsent(sectionName, new LinkedHashMap<>());
```

Comment:
To me this `putIfAbsent` is not necessary as a few lines ago the condition `if this.sessionLevel.containsKey(sectionName)` was just checked. Therefore I think we can avoid the duplicate checking here.